client/src/util/api_util.js

```diff
@@ -15,10 +15,40 @@ export function getFriendsPolls(callback) {
   });
 }
 
-export function getUserPolls() {
-  return fetch('/polls', {
-    method: 'GET',
-  }).then((res) => res.json());
+export async function getUserPolls() {
+  const response = await fetch('/polls', { method: 'GET' });
+  const body = await response.json();
+
+  if (response.status !== 200) {
+    throw Error(body.error);
+  }
+  return body;
+}
+
+export async function deleteUserPoll(_id) {
+  const response = await fetch(`/polls/${_id}`, { method: 'DELETE' });
+  const body = await response.json();
+
+  if (response.status !== 200) {
```

Comment:
Side note: 
  - Another way to write this (in case it's not exactly `200` but still ok) is `response.ok`. It returns a `boolean`.
    - But this is fine since that's how we do it in the rest of the app, and we're only returning `200`s

Reply:
Yeah, I don't think I wrote the fetch functions the best way possible. `response.ok` seems better.